scripts/merge-configs.js

```diff
@@ -47,3 +47,42 @@ hexo.on('generateBefore', function() {
   require('./injects')(hexo);
 
 });
+
+hexo.on('generateBefore', function() {
```

Comment:
You created a new `generateBefore` listener, but what is the order of it and the merge configuration?

`hexo.theme.config.reminder` can also be from `next.yaml`

Reply:
I see.

> When the EventEmitter object emits an event, all of the functions attached to that specific event are called synchronously.

Reply:
Maybe hexo on exit is better.

Reply:
Good idea. But `http.get` is synchronous, `on exit` will take more time.
Maybe this one:
https://github.com/iissnan/hexo-theme-next/blob/620b1e829eb8b6fd72426f3009866b79d8ee2e7b/scripts/merge-configs.js#L22-L30